src/app/core/services/basic-rundown-state.service.ts

```diff
@@ -84,6 +95,10 @@ export class BasicRundownStateService implements OnDestroy {
     return this.basicRundownsSubject.subscribe(consumer)
   }
 
+  public subscribeToLoading(consumer: (isLoading: boolean) => void): SubscriptionLike {
```

Comment:
I think it would be a good idea that you follow the same naming convention that you did for all your other "consumer" parameters. So this parameter would be called `onLoad`

Reply:
A second read through and I realise that your Subject doesn't only emit on load, so my naming suggestion would not be accurate.
I do however, still prefer that we don't call it `consumer` :)

Reply:
I also thought about changing it so `onX`. I just couldn't find a proper name, and thought it was a bit overkill to split it into two subscriptions.